packages/es-components/src/components/controls/buttons/DropdownButton.js

```diff
@@ -135,6 +148,18 @@ function arrowMovement(node) {
 }
 
 function DropdownButton(props) {
+  const {
+    rootClose,
+    children,
+    manualButtonValue,
+    styleType,
+    inline,
+    split,
+    ...otherProps
+  } = props;
+
```

Comment:
It looks like `props` only gets used one other time, and it's just to pull out some other values. It seems like we should do this destructuring as function parameters rather than keeping an extra `props` variable around.